packages/web3-eth/package.json

```diff
@@ -36,12 +36,12 @@
 		"test": "jest --config=./test/unit/jest.config.js",
 		"test:coverage:unit": "jest --config=./test/unit/jest.config.js --coverage=true --coverage-reporters=text",
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
-		"test:e2e:mainnet": "jest --config=./test/e2e/jest.config.js --forceExit",
-		"test:e2e:sepolia": "jest --config=./test/e2e/jest.config.js --forceExit",
+		"test:e2e:mainnet": "jest --config=./test/e2e/jest.config.js",
+		"test:e2e:sepolia": "jest --config=./test/e2e/jest.config.js",
```

Comment:
@luu-alex @jdevcs  these tests from web3-eth package seems to be always skipped, even in the release builds, maybe we should just delete them altogether? ie. see https://github.com/web3/web3.js/actions/runs/11710089473/job/32616617212

```
> web3-eth:"test:e2e:sepolia"

$ jest --config=./test/e2e/jest.config.js --forceExit

Test Suites: 1 skipped, 0 of 1 total
Tests:       1 skipped, 1 total
```